pkgs/tools/audio/tts/default.nix

```diff
@@ -12,25 +12,24 @@
 #
 # If you upgrade from an old version you may have to delete old models from ~/.local/share/tts
 # Also note that your tts version might not support all available models so check:
-#   https://github.com/coqui-ai/TTS/releases/tag/v0.0.15.1
+#   https://github.com/coqui-ai/TTS/releases/tag/v0.1.2
 #
 # For now, for deployment check the systemd unit in the pull request:
 #   https://github.com/NixOS/nixpkgs/pull/103851#issue-521121136
 
 python3Packages.buildPythonApplication rec {
   pname = "tts";
```

Comment:
I find the new test suite quite slow. Can we exclude some of the slow tests somehow?

Reply:
On my desktop the build including tests completes in under 10 minutes, which I personally think is bearable, to make sure we didn't miss anything.

Maybe it is time to create a module and disable the tests for endusers, just like we did for home assistant? Ultimately if the package isn't cached it is usually due to a tensorflow or pytorch issue, so I hardly think the tts build time matters alot.

Reply:
Ok. It took 30 minutes on mine.